setup.py

```diff
@@ -34,6 +34,10 @@ def run(self, *args, **kwargs):
     ]),
     use_scm_version=True,
     setup_requires=['setuptools_scm'],
+    author="Azure Functions team at Microsoft Corp.",
+    author_email="azurefunctions@microsoft.com",
+    keywords="azure functions azurefunctions python serverless workflows durablefunctions",
```

Comment:
We have two azure functions in this line. Is this ok? Other than this one, all the other changes look good to me! :)

Reply:
thanks! That second "azurefunctions" is distinct from "azure functions" because they represent different keywords: "azure functions" is 2 keywords, and "azurefunctions" is 1 keyword. These are just labels that PyPI will use to categorize our package, so we're just trying to capture different spellings of the product :)